docker-compose.yml

```diff
@@ -12,14 +12,15 @@ services:
     image: redis:alpine
     ports:
       - "6379:6379"
-  selenium-firefox:
+  selenium:
     image: b4handjr/selenium-firefox
-    volumes:
-      - .:/send
-    working_dir: /send
-    expose:
-      - "4444"
     ports:
       - "5900"
       - "4444:4444"
     shm_size: 2g
+  integration-tests:
+    build: ./test/integration
```

Comment:
So you can build another image even though you pull an image? That's pretty neat.